pkg/engine/imageVerify.go

```diff
@@ -339,6 +340,11 @@ func (iv *imageVerifier) verifyImage(
 	}
 
 	if len(imageVerify.Attestors) > 0 {
+		if !datautils.SliceContains(imageVerify.ImageReferences, imageInfo.String()) {
```

Comment:
`imageVerify.ImageReferences` could be wildcard matches right?

Reply:
Yes, the cli test also failed.  Pushed the fix.